index.js

```diff
@@ -223,20 +227,36 @@ const artists = [
  * it will return `The artist at index 0 is Amedeo Modigliani`.
 */
 function getArtistByIndex(array, index) {
-    /* code here */
+    return `The artist at index ${array[index].id} is ${array[index].name}.`;
   }
   
+console.log(getArtistByIndex(artists,0));
   /**
 
 
-/* Task 4: Create a function called get20s() that takes data as an argument and returns an array with names of artists who were born in and died in 20th century (1900-2000) example born in 1901 and died in 1959 - included / born in 1889 and died in 1925 not included - should return ["Salvador Dali", "Frida Kahlo"]*/
+/* Task 4: Create a function called get20s() that takes data as an argument and returns an array with 
+names of artists who were born in and died in 20th century (1900-2000) example born in 1901 and died in 1959 - included / born in 1889 and died in 1925 not included
+- should return ["Salvador Dali", "Frida Kahlo"]*/
 
-function get20s(/* Code here */){
-
-  /* Code here */
+function get20s(array){
+ let names=[];
+for (let k of array)
+{
+  let holder=[]
```

Comment:
Great job on using the split method. You can utilize the `+` sign here to [parse a string to an integer](https://www.javascripttutorial.net/javascript-unary-operators/)

```js
let holder = +k.years.split("-")
if(holder[0]>=1990){
 ...
}
```